.travis.yml

```diff
@@ -1,21 +1,39 @@
-language: node_js
-node_js:
-    - "12"
-branches:
-    only:
-        - master
-        - develop
-
-addons:
-    sonarcloud:
-        organization: "usdot-jpo-sdc"
-
-before_script:
-  - npm install -g @angular/cli
-
-env:
-  - TEST_DIR=webportal/webapp/
-
-script:
-    - cd $TEST_DIR && npm install && npm run test-headless
-    - cd ../../ && sonar-scanner -X -D sonar.login=$SONAR_TOKEN
+matrix:
+  include:
+    - language: node_js
+      node_js: "12"
+      branches:
+        - only:
+          - master
+          - develop
+      addons:
+        - sonarcloud:
+          - organization: "usdot-jpo-sdc"
+      before_script:
+        - npm install -g @angular/cli
+      env:
+        - TEST_DIR=webportal/webapp/
+      script:
+        - cd $TEST_DIR && npm install && npm run test-headless
+        - cd ../../ && sonar-scanner -X -D sonar.login=$SONAR_TOKEN
+    - langauge: python
+      python: 3.6
+      branches:
+        - only:
+          - master
+          - develop
+      addons:
+        - sonarcloud:
+          - organization: "usdot-jpo-sdc"
+      install:
+        - pip install boto3
```

Comment:
Can you just do a pip install using the requirements.txt file?

Reply:
That's what I tried at first and I couldn't get it to work. I think I may have been doing it wrong though. I tried pip installing the file in a before_script and it just didn't install anything. Lemme try one thing...